README.md

```diff
@@ -1,34 +1,56 @@
 # cltdevs meetup manifesto
 
 ## Mission statement
-We grow Charlotte's tech community by bringing people together.
+We are committed to the Charlotte tech community's growth and sustainability by fostering relationships between individuals and organizations, cultivating individual talents and organizations, and discover and build the missing elements of a thriving tech community.
```

Comment:
I agree that a reduction in words makes sense...but our mission statement should probably be pretty solid...its the core thing that everything flows from...`bringing people together` was the part that i felt needed the most replacing as it doesnt really describe anything while also limiting our potential. As I was trying to figure this sentence out, the piece that I felt the strongest about was the `discover and build the missing elements of a thriving tech community` aspect as I feel like that's kinda the core of what we are doing here...finding pieces that are missing and then finding a way to build them...

Suggestions wanted.